doc/ref_model/chapters/chapter04.md

```diff
@@ -288,7 +288,7 @@ Table 4-12 shows capabilities related to resources allocation
 | e.vim.cap.001 | Virtual Compute allocation | Yes/No | Capability to allocate virtual compute resources  to VNFC |
 | e.vim.cap.002 | Virtual Storage allocation | Yes/No | Capability to allocate virtual storage resources  to VNFC |
 | e.vim.cap.003 | Virtual Networking resources allocation | Yes/No | Capability to allocate virtual networking resources  to VNFC |
-| e.vim.cap.004 | Multi-tenant isolation | Yes/No | Capability to isolate resources between tenants |
+| e.vim.cap.004 | Multi-tenant isolation | Yes/No | Capability to isolate resources between tenants <br> **Note:** In a Kubernetes based infrastructure within one Kubernetes cluster multitenacy is provided only in resource management level. Isolation of execution environments requires separate Kubernetes clusters. |
```

Comment:
Should this not be "separate Kubernetes worker nodes" rather than "separate Kubernetes clusters"?

Reply:
Humm, it depends on the details we would like to add here. Network separation is not solved even in case of different worker nodes. What if I add "separate Kubernetes clusters or worker nodes"?

Reply:
Yes true. Yes, I think your suggestion would capture all "types" of resource isolation.